### PR TITLE
Update wdqs updater to 0.3.84, and reduce memory leaks (again)

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    command: /runUpdate.sh
+    depends_on:
+      - wdqs
+    networks:
+      default:
+        aliases:
+          - wdqs-updater.svc
+    environment:
```
[the remark this answers is not in the document]
Comment (toban) [reply docker-compose.yml:48]: done

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      default:
+        aliases:
+          - api.svc
+
+  wdqs-updater:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    command: /runUpdate.sh
```
[the remark this answers is not in the document]
Comment (toban) [reply docker-compose.yml:41]: done

**Comment by addshore** [comment]
Paperish trail about why we decided that it would be fine to update the wdqs version just for the updater:

```
1:17 PM <addshore> I have a question back at you! If I were to start using the 0.3.84 wdqs updater code, from 0.3.6 any ideas what ight change in rdf output?
1:17 PM <addshore> or where to look to see what might change?
1:17 PM <+gehel> that's a question for dcausse 
1:18 PM <+dcausse> addshore: nothing
1:18 PM <+dcausse> regarding the RDF itself I mean
1:18 PM <addshore> okay, so the only changes in wdqs version numbers really are the wey the internal indexing happens?
1:19 PM <+dcausse> yes and bugfixes
1:19 PM <+dcausse> I mean at the query side of things
1:19 PM <addshore> epic
1:20 PM <+dcausse> since it's a multi-module project the updater might get a version bump even-though nothing actually changed there
1:20 PM <addshore> yup,, niceeeeee
1:20 PM <+dcausse> 0.3.6 is quite old tho :)
1:20 PM <addshore> so for this wbstack / wbaas thing we are gonig to base a "new" updater on 0.3.84 or something but keep 0.3.6 for the actually suery service for now
1:21 PM <+dcausse> ok
1:21 PM <+dcausse> lemme know if you run into troubles
1:22 PM <+dcausse> but generally speaking we never change the RDF output, it's part of the stable interface policy
1:24 PM <addshore> ack!
```

File: .env [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## Image Configuration
+WIKIBASE_IMAGE_NAME=wikibase/wikibase:1.35.2-wmde.1
```
Comment (addshore) [inline .env:2]: This is fine for now but at some point we might want to try and hook this up to the actual version of mw runnning as part of wbstack.

**Pull request merged**